src/olympia/addons/tests/test_serializers.py

```diff
@@ -536,6 +537,19 @@ def test_no_license(self):
         assert result['id'] == self.version.pk
         assert result['license'] is None
 
+    def test_file_webext_permissions(self):
```

Comment:
https://github.com/mozilla/addons-server/pull/4651/commits/103e008e18c81d7066a0f9bcb2f6928102bc74f8 (and fixes for the other comments)